dev/com.ibm.websphere.appserver.features/visibility/public/jaxb-2.3/com.ibm.websphere.appserver.jaxb-2.3.feature

```diff
@@ -5,6 +5,7 @@ singleton=true
 IBM-App-ForceRestart: uninstall, \
  install
 IBM-API-Package: \
+  javax.activation; type="spec"; require-java:="9", \
```

Comment:
since this feature will always include javax.activation, this part doesn't need to be conditional on Java 9, so you can remove the `require-java` clause